packages/jest-dom-mocks/src/promise.ts

```diff
@@ -12,7 +12,7 @@ export default class Promise {
     }
 
     jest.useFakeTimers();
-    global.Promise = FakePromise;
+    global.Promise = FakePromise as any;
```

Comment:
Why do we need to `as any` this now but not before?

Reply:
I had the same question and I tried to remove it.

<img width="1273" alt="image" src="https://user-images.githubusercontent.com/4728325/76449043-bd989780-63a1-11ea-8136-4745a4370e28.png">

Reply:
`@types/node` `13` bump. This was introduced in that. See https://github.com/Shopify/quilt/pull/1316#issuecomment-597284951